Sources/HiPMobile/DataAccessLayer/EFCoreTransaction.cs

```diff
@@ -34,22 +34,22 @@ public EFCoreTransaction(AppDatabaseContext db)
         {
             this.db = db ?? throw new ArgumentNullException(nameof(db));
             DataAccess = new EFCoreDataAccess(db);
-            transaction = db.Database.BeginTransaction();
+                // transaction = db.Database.BeginTransaction();
```

Comment:
Can you tell me why you do not use a database transaction anymore?

Reply:
This was a workaround for a database issue that will be fixed with #245. Thus the change to `EFCoreTransaction.cs` shouldn't be included in this PR for HIPM-943.